### PR TITLE
Adding more content to community README to help guide folks

### PR DESCRIPTION
**Pull request by aburdenthehand**
Fleshing out our README file with useful links to getting started, contributing guide, sig-release, community meeting, adopters file, and probably something else too.

I've added a lot of links, which have the terrible fate of changing sometimes, so I've tried to be mindful of this and link only to more robustly located things. 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fabiand** [comment]
/approve

@dhiller can you give this a second set of eyes?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
 # Community
 
-Here you can find community content around KubeVirt.
+Welcome to the KubeVirt Community repo!
+Here you will find all things community, including our [membership policy](membership_policy.md), [project governance](https://github.com/kubevirt/community/blob/main/GOVERNANCE.md), charters for our SIGs (Special Interest Groups), [upcoming events](https://github.com/kubevirt/community/wiki/Events), and more.
```
Comment (jean-edouard) [inline README.md:4]:
```suggestion
Here you will find all things community, including our [membership policy](membership_policy.md), [project governance](GOVERNANCE.md), charters for our SIGs (Special Interest Groups), [upcoming events](https://github.com/kubevirt/community/wiki/Events), and more.
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+
+If you're looking for the main code base, you can find it at [kubevirt/kubevirt](
```
Comment (jean-edouard) [inline README.md:6]: Missing URL

**Review by dhiller — CHANGES_REQUESTED** [review]
As Jed said, we can shorten the GOVERNANCE url and add the k/kubevirt url.

**Comment by aburdenthehand** [comment]
@jean-edouard Thanks for the catch!
I have updated both as requested.

**Review by jean-edouard — APPROVED** [review]

**Pull request merged**